src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatController.ts

```diff
@@ -221,6 +251,16 @@ export class TerminalChatController extends Disposable implements ITerminalContr
 			// TODO: ?
 			variables: { variables: [] },
 		};
+		// TODO: fix requester usrname, responder username
+		this._model?.initialize({ id: this._accessibilityRequestId, requesterUsername: 'userGesture', responderUsername: 'terminal' }, undefined);
```

Comment:
not sure why I have to call this, need to investigate why `startSession` doesn't do this for us